python/ui/test/test_dropdown_option_select.py

```diff
@@ -0,0 +1,23 @@
+from python.common.basebrowser import BrowserExample
+from python.ui.method.dropdown_page import Dropdown
+from python.ui.method.internet_home_page import Homepage
+
+
+class TestDropdown:
+
```

Comment:
This may be an extra return, I cant remember the PIP standard on class spacing.

Reply:
Spacing is acceptable, since PIP suggests one space between class and method declarations.